### PR TITLE
Add features and modofy documents

### PR DESCRIPTION
**Pull request by Yujian-Fu**
Add features: help, report_person
Modify documents: DG, UG, Anout_Us, PPP of Yujian

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -172,56 +178,3 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
...
-[appendix]
-== User Stories
-
-Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
-
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
-|=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-
-|=======================================================================
```
Comment (JKOK005) [inline docs/DeveloperGuide.adoc]: Why remove this part

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -172,56 +178,3 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
...
-```java
-java -jar <application.jar>
-```
-
-. Initial launch
-
-.. Download the jar file and copy into an empty folder
-.. Run the command above
-
-   Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```
Comment (JKOK005) [inline docs/DeveloperGuide.adoc]: This part is important to explain to the user how to launch the app

**Review by JKOK005 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (Yujian-Fu) [reply docs/DeveloperGuide.adoc]: Already restored this part

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (Yujian-Fu) [reply docs/DeveloperGuide.adoc]: I wrote two classes for help command: CommandList that contains strings for display. CommandListMessage as the return result. When a input string "help" is detected, the CommandRouter will call HelpCommand to handle it.

**Review by JKOK005 — APPROVED** [review]

**Pull request merged**